frontend/src/components/KeyMeasureDiagnostic/ExpeModals/ExpeVegetarian.vue

```diff
@@ -490,22 +603,89 @@
+                      :for="`recipe-document-${item.value}`"
+                      v-if="expe[`hasUsedRecipeDocuments${item.value}`]"
+                    >
+                      Précisez, quel livret de recette avez vous utilisé ?
+                    </label>
+                    <v-textarea
+                      v-if="expe[`hasUsedRecipeDocuments${item.value}`]"
+                      solo
+                      :id="`recipe-document-${item.value}`"
+                      v-model="expe[`recipeDocument${item.value}`]"
+                      rows="2"
+                      hide-details="auto"
+                    ></v-textarea>
+
                     <v-divider class="mt-6"></v-divider>
                     <!-- Formation -->
                     <v-checkbox hide-details="auto" class="my-4" v-model="expe[`training${item.value}`]">
                       <template v-slot:label>
                         <span class="body-2 grey--text text--darken-3 font-weight-medium">
-                          Votre établissement a-t-il mis en place une formation spécifique des cuisiniers ou
-                          gestionnaires sur les menus végétariens ?
+                          Mon établissement a mis en place une formation spécifique des cuisiniers ou gestionnaires sur
+                          les menus végétariens
                         </span>
                       </template>
                     </v-checkbox>
+
+                    <label
+                      class="body-2 grey--text text--darken-3 font-weight-medium d-block mb-4"
+                      :for="`training-type-${item.value}`"
+                      v-if="expe[`training${item.value}`]"
+                    >
+                      Précisez, quel type de formation ?
+                    </label>
+                    <v-textarea
+                      v-if="expe[`training${item.value}`]"
+                      solo
+                      :id="`training-type-${item.value}`"
+                      v-model="expe[`trainingType${item.value}`]"
+                      rows="2"
+                      hide-details="auto"
+                    ></v-textarea>
+
+                    <v-divider class="my-6"></v-divider>
+                    <!-- Difficulties -->
+                    <label
+                      class="body-2 grey--text text--darken-3 font-weight-medium"
+                      :for="`difficulties-daily-option-${item.value}`"
+                    >
+                      Quels sont les principaux freins rencontrés à la mise en place de l’option végétarienne
+                      quotidienne ?
+                    </label>
+                    <v-radio-group
+                      :id="`difficulties-daily-option-${item.value}`"
+                      v-model="expe[`difficultiesDailyOption${item.value}`]"
+                      class="mb-2"
+                    >
+                      <v-radio v-for="option in difficultiesOptions" :value="option.value" :key="option.value">
```

Comment:
The question here is in plural 'principaux freins' - should the user be allowed to choose multiple?

Reply:
Fix ici : https://github.com/betagouv/ma-cantine/pull/1382/commits/cd692cac23c3fd5c717b6b82e6afb03d82775307